StaticAnalyzer/views/android/static_analyzer.py

```diff
@@ -89,12 +91,17 @@ def static_analyzer(request, api=False):
             app_dic['dir'] = settings.BASE_DIR  # BASE DIR
             app_dic['app_name'] = filename  # APP ORGINAL NAME
             app_dic['md5'] = checksum  # MD5
+            # APP DIRECTORY
             app_dic['app_dir'] = os.path.join(settings.UPLD_DIR, app_dic[
-                                              'md5'] + '/')  # APP DIRECTORY
+                                              'md5'] + '/')
             app_dic['tools_dir'] = os.path.join(
                 app_dic['dir'], 'StaticAnalyzer/tools/')  # TOOLS DIR
             logger.info('Starting Analysis on : %s', app_dic['app_name'])
 
+            # Will inject the different pattern strategy
+            # when it it will be requested
+            match_command = MatchCommand()
```

Comment:
move this to https://github.com/Arenash1313/Mobile-Security-Framework-MobSF/blob/master/StaticAnalyzer/views/android/code_analysis.py#L29